app/org/sagebionetworks/bridge/models/schedules/ActivityScheduler.java

```diff
@@ -152,6 +161,14 @@ protected boolean shouldContinueScheduling(ScheduleContext context, DateTime sch
         return boundaryNotMet;
     }
     
+    private boolean scheduledOnAfterSequencePeriod(ScheduleContext context, DateTime scheduledTime, int currentCount) {
+        // Don't know why you'd specify a sequence, then force N number of activities to be returned, but the 
+        // minimum activities setting is part of the public API and it would be unexpected if it didn't work.
+        
+        boolean minMet = context.getMinimumPerSchedule() == 0 || currentCount >= context.getMinimumPerSchedule();
+        return sequenceEndsOn != null && !scheduledTime.isBefore(sequenceEndsOn) && minMet;
```

Comment:
As per IM conversation, this behavior is counter-intuitive and is not consistent with how schedule.endsOn works. We should remove the interaction between minimumPerSchedule and sequencePeriod.